features/mbedtls/inc/psa/crypto.h

```diff
@@ -61,20 +61,7 @@ extern "C" {
  * @{
  */
 
-#if defined(PSA_SUCCESS)
```

Comment:
shouldn't this change be done in mbedtls (new version would update this) ?

Reply:
yes but, to not break the compilation i must include this chnage

Reply:
Yes, this change should be done upstream first via a new PR to https://github.com/ARMmbed/mbed-crypto

Reply:
@Patater FYI https://github.com/ARMmbed/mbed-crypto/pull/5

Reply:
The necessary changes will come in through https://github.com/ARMmbed/mbed-os/pull/9463

Reply:
Than I suggest removing this file from this PR and wait for #9463 to be merged first.